docs/src/main/paradox/release-notes/1.1.md

```diff
@@ -0,0 +1,23 @@
+# Alpakka Kafka 1.1
+
+## Alpakka Kafka 1.1 to upgrade the Apache Kafka client to 2.1.1
```

Comment:
Why isn't it a patch version? 1.0.1? Update of the Kafka client from 2.1.0 to 2.1.1 seems (from the version number) to be a compatible update.

Reply:
Changed to 1.0.1 now.